test/scale/manifest/single-az.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
We'll either have to modify the `azure-cluster-up.sh` script to search this directory, or move this file to `test/e2e/manifest`.